src/Services/membersService.js

```diff
@@ -1,31 +1,31 @@
 import axios from "axios";
 import getAuthorizationHeader from "./privateApiService";
 
-const URL = "http://ongapi.alkemy.org/api";
+const URL = process.env.REACT_APP_API_URL_MEMBERS;
 const authorizationHeader = getAuthorizationHeader();
 
 const getMembers = () => {
-  const response = axios.get(`${URL}/members`, authorizationHeader);
+  const response = axios.get(`${URL}`, authorizationHeader);
```

Comment:
the config that you pass to axios should look something like this: 
```
const config = {
    headers: { Authorization: `Bearer ${token}` }
};
```
and the authorization header its only the header, that is why you should make something like:
```
{
    headers: authorizationHeader
}
```

Reply:
you can create this object at the beginning, instead of const authorizationHeader = getAuthorizationHeader it should be something like:
```
const axiosConfig = { headers: getAuthorizationHeader() }
```
and then:
```
const response = axios.get(`${URL}`, axiosConfig);
```

Reply:
I know that this is not scope of this user story but if you can do it it will be great!